docs/book/src/getting_started/README.md

```diff
@@ -0,0 +1,120 @@
+# Getting Started
+
+There are two basic paths to getting started with Leptos:
+
+1. Client-side rendering with [Trunk](https://trunkrs.dev/) - a great option if you just want to make a snappy website with Leptos, or work with a pre-existing server or 3rd-party API's.
+2. Full-stack / Universal server-side rendering with [`cargo-leptos`](https://github.com/leptos-rs/cargo-leptos) - a great option for building a CRUD-style website or a web-app with Rust powering both frontend and backend. Universal apps rely on either Actix-web or Axum on the server (so you may want to keep their docs handy, too), or you can even go Universal WASM/WASI with the Leptos and Axum integration (more on these options later).
+
+In Part 1 of this book, we'll start with Client-side rendering  Leptos sites and building reactive UI's using `Trunk` to serve our JS+Wasm bundle to the browser.
```

Comment:
small one but I usually go for "client-side" lowercases and "JS and WASM" or "JS & WASM"

there's also a double space between "rendering" and "Leptos"